.svglintrc.js

```diff
@@ -163,6 +163,44 @@ module.exports = {
               }
             }
           },
+          function(reporter, $, ast) {
+            reporter.name = "icon-path";
+
+            const iconPath = $.find("path").attr("d");
+            if (!updateIgnoreFile && isIgnored(reporter.name, iconPath)) {
+              return;
+            }
+
+            const { segments } = parsePath(iconPath);
+
+            const lowerCommand = ['m', 'l', 'h', 'v'];
+            const upperCommand = ['M', 'L', 'H', 'V'];
+            const commands = [...lowerCommand, ...upperCommand];
+            const getInvalidSegments = ([command, coord1, coord2, ...rest], index) => {
+              if (commands.includes(command)) {
+                if (lowerCommand.includes(command) && coord1 === 0 && coord2 === 0) {
+                  return true;
+                }
+                if (index > 0) {
+                  const [prevCoord2, prevCoord1, ...rest] = [...segments[index - 1]].reverse();
+                  if (upperCommand.includes(command) && coord1 === prevCoord1 && coord2 === prevCoord2) {
+                    return true;
+                  }
+                }
+              }
+            };
+            const invalidSegments = segments.filter(getInvalidSegments);
+
+            if (invalidSegments.length) {
+              invalidSegments.forEach(([command, coord1, coord2]) => {
+                const readableSegment = `${command}${coord1} ${coord2}`;
```

Comment:
Something I haven't realized before, is that the H and V commands ony has a number, so something like this would be necessary:

```suggestion
                let readableSegment = `${command}${coord1}`;
                if (coord2) {
                   readableSegment += ` ${coord2}`;
                }
```